### PR TITLE
explicitly copy some scripts needed by the regedit module

### PR DESCRIPTION
**Pull request by zanderz**
@keybase/react-hackers 
Making a test build now

### CODE REVIEW

File: shared/actions/kbfs/index.platform.desktop.js [hunk 1]
```diff
@@ -176,43 +176,50 @@ function* installFuseSaga(): SagaGenerator<any, any> {
...
-            entry.data.values.BundleCachePath
-          ) {
-            if (fs.existsSync(entry.data.values.BundleCachePath.value)) {
-              var modifyPath = entry.data.values.ModifyPath.value
-              // Remove double quotes - won't work otherwise
-              modifyPath = modifyPath.replace(/"/g, '')
-              // Remove /modify and send it in with the other arguments, below
-              modifyPath = modifyPath.replace(' /modify', '')
-              resolve(modifyPath)
+    try {
```
Comment (zanderz) [inline shared/actions/kbfs/index.platform.desktop.js]: This just was wrapped in try/catch. I don't think it helped anything after all but it seems harmless.

File: shared/desktop/package.js [hunk 2]
```diff
@@ -188,6 +188,19 @@ function log(plat, arch) {
         process.exit(1)
       }
     })
+    if (plat === 'win32') {
```
Comment (zanderz) [inline shared/desktop/package.js]: This stuff had to happen in the callback, after the main packaging was finished

File: shared/desktop/package.js [hunk 3]
```diff
@@ -188,6 +188,19 @@ function log(plat, arch) {
...
+      let packageOutDir = outDir
```
Comment (chrisnojima) [inline shared/desktop/package.js]: this shouldn't be a function named 'log' anymore if its doing more than logging

File: shared/desktop/package.js [hunk 4]
```diff
@@ -188,6 +188,19 @@ function log(plat, arch) {
...
+      if (packageOutDir === '') packageOutDir = desktopPath(`release/${plat}-${arch}`)
+      let regeditVbsDir = packageOutDir + `/Keybase-${plat}-${arch}/node_modules/regedit/vbs`
```
Comment (chrisnojima) [inline shared/desktop/package.js]: const let regeditVbsDir = ${packageOutDir}/Keybase-${plat}-${arch}/node_modules/regedit/vbs`

File: shared/desktop/package.js [hunk 5]
```diff
@@ -188,6 +188,19 @@ function log(plat, arch) {
...
+      fs
+        .ensureDir(regeditVbsDir)
+        .then(() => {
+          fs.copy('node_modules/regedit/vbs', regeditVbsDir)
+        })
+        .catch(err => {
+          console.error(err)
```
Comment (chrisnojima) [inline shared/desktop/package.js]: should process.exit(1) in this case

**Pull request merged**